CRM/Admin/Form/Setting.php

```diff
@@ -147,6 +152,9 @@ public function buildQuickForm() {
 
       }
     }
+    if (!empty($setStatus)) {
+      CRM_Core_Session::setStatus("Some fields are loaded as 'readonly' as they have been set (overridden) in civicrm.settings.php.", '', 'info', array('expires' => 0));
```

Comment:
@jitendrapurohit This string is missing `ts()` for translation. Could you do a follow-up PR?

Reply:
Done - https://github.com/civicrm/civicrm-core/pull/11802. Thanks @mlutfy